src/main/kotlin/ch/komunumo/server/business/configuration/control/ConfigurationService.kt

```diff
@@ -33,13 +36,42 @@ object ConfigurationService {
 
     private fun loadConfiguration(): Properties {
         val properties = Properties()
-        val file = Paths.get(System.getProperty("user.home"), ".komunumo", configurationFilename).toFile()
+        val filePath = System.getenv(envVariableName) ?: ""
+        val file: File = configFile(filePath)
+
+
         if (file.exists()) {
+            ConfigurationService.logger.info { "Using configuration: " + file.toString() }
+            println("Using configuration: " + file.toString())
+
             FileInputStream(file).use { stream -> properties.load(stream) }
+        } else {
+            ConfigurationService.logger.info { "No configuration file found under " + file.toString() }
+            println("No configuration file found under " + file.toString())
```

Comment:
Komunumo will not work without a configuration file. Maybe it is a good idea to stop Komunumo if there was no configuration file found. Maybe with throwing a `FileNotFoundException`.